README.md

```diff
@@ -15,5 +15,12 @@ The annotation package which has no dependencies.
 
 Import it into your pubspec `dependencies:` section.
 
+## [example]
+
+A simple example showing how to setup and use [json_serializable] and
```

Comment:
I'm confused, where is the example?

Reply:
See https://github.com/dart-lang/json_serializable/tree/example (although the link is broken because this isn't on master yet)